packages/fixie/src/voice.ts

```diff
@@ -147,6 +163,7 @@ export class VoiceSession {
     }
     this.started = true;
     this.maybePublishLocalAudio();
+    this.changeState(VoiceSessionState.IDLE);
```

Comment:
the socket might not yet be connected here, I think this needs to happen in maybePublishLocalAudio

Reply:
Good catch, done